src/platforms/unreal/configuration/setup-crashreporter.mdx

```diff
@@ -7,57 +7,55 @@ redirect_from:
+
+</Alert>
+
+## UE Crash reporter
+
+The UE Crash Reporter is provided out of the box by Epic Games. It is able to collect relevant
 information about the crash and send it to a service like Sentry to process.
 
 Adding it to your game means that in case a crash happens, the following dialog is displayed
 to the user:
 
-![The UE4 Crash Reporter](ue4-crash-reporter.png)
+![The UE Crash Reporter](ue-crash-reporter.png)
 
-### Include the UE4 Crash Reporter
+### Include the UE Crash Reporter
 
 You can add the _Crash Reporter Client_ to your game in your _Project Settings_.
 
 The simplest way is to search the option: `Crash Reporter`:
 
-![Include UE4 Crash Reporter](ue4-include-crash-reporter.png)
+![Include UE Crash Reporter](ue-include-crash-reporter.png)
 
 The option is located under _Project > Packaging_ menu, then select _show advanced_ followed by
 checking the box for: `Include Crash Reporter`.
 
 ## Debug information
 
-To get the most out of Sentry, crash reports are required to include debug information.
+To get the most out of captured crash reports [debug symbols need to be uploaded to Sentry](#upload-debug-symbols).
 In order for Sentry to be able to process the crash report and translate
 memory addresses to meaningful information like function names, module names
-and line numbers, the crash itself must include debug information and also, [symbols need
-to be uploaded to Sentry](#upload-debug-symbols).
-
-### Include Debug Information
-
-![Include Debug Files](ue4-include-debug-files.png)
-
-The option is also located under _Project > Packaging_ menu, then select _show advanced_ followed by
-checking the box for: `Include Debug Files`.
+and line numbers, the crash itself must be associated with the corresponding debug information first.
 
 ## Configure the Crash Reporter Endpoint
 
-Now that the _Crash Reporter_ and _Debug Files_ are included, UE4 needs to know where to send the
+Now that the _Crash Reporter_ and _Debug Files_ are included, UE needs to know where to send the
 crash. For that, we add the Sentry _Unreal Engine Endpoint_ from the _Client Keys_ settings page to game's configuration file. This will
 include which project within Sentry you want to see the crashes arriving in real time.
 That's accomplished by configuring the `CrashReportClient` in the _DefaultEngine.ini_ file. Changing the engine is necessary for this to work.
```

Comment:
```suggestion
Now that the _Crash Reporter_ and _Debug Files_ are included, UE needs to know where to send the
crash. For that, we need to add the Sentry _Unreal Engine Endpoint_ from the _Client Keys_ settings page to the game's configuration file. This will include whichever project within Sentry you want to see real-time crashes for. That's accomplished by configuring the `CrashReportClient` in the _DefaultEngine.ini_ file. You'll need to change the engine to do this to work.
```